fili-core/src/main/java/com/yahoo/bard/webservice/logging/blocks/BardQueryInfo.java

```diff
@@ -32,8 +32,8 @@
     private final AtomicInteger factCacheHitCount = new AtomicInteger();
     private final AtomicInteger factPutErrorsCount = new AtomicInteger();
     private final AtomicInteger factPutTimeoutsCount = new AtomicInteger();
-    private Map<String, BardCacheInfo> putFailuresMap = new HashMap<>();
-    private Map<String, BardCacheInfo> readFailuresMap = new HashMap<>();
+    private Map<String, BardCacheInfo> putCacheStatsMap = new HashMap<>();
```

Comment:
Per discussion, remove putCacheStats (because cache put happens after return so we'll never have cache put data)